frontend/src/index.pug

```diff
@@ -50,6 +50,8 @@ html
             )
               strong User Guide - {{ repoSenseVersion }}
             span ) on {{ creationDate }}
+          .report-generation-time(style="visibility:hidden;font-size: 0;")
```

Comment:
perhaps only `display: none`?

Reply:
Thanks for looking through! I agree; `display: none` is simpler and achieves the same purpose.